js/index.js

```diff
@@ -1,7 +1,364 @@
+const User_List = document.getElementById('user-list');
+let USER_COUNT =0;
+
+// 해당 서버에 데이터 가져와 처리
+function Response_Api(){
+  fetch('https://js-todo-list-9ca3a.df.r.appspot.com/api/users')
```

Comment:
반복 사용되는 변하지 않는 내용은 상수로 분리해주시면 더 보기 좋은 코드가 될거 같습니다!
```suggestion
  const BASE_URL = 'https://js-todo-list-9ca3a.df.r.appspot.com/'
  fetch(`${BASE_URL}api/users`);
```